contracts/Cabell.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.5.2;
+
+/**
+ * @title Cabell interface
+ * @dev Quit yer bitchin and get after it
```

Comment:
change this to `@notice` and have `@dev` note that because `doTheWork` accepts no arguments, all data it operates on must be deployed in the contract code or accessed via an external call

Reply:
Will fix.